### ISSUE TITLE
[jdocs] Joomla! CMS Versions table looks incorrect / outdated

### ISSUE THREAD

**Issue by marcwrobel** [joomla/joomla-websites#1842]
4.x version documentation on https://docs.joomla.org/Joomla!_CMS_versions is I think incorrect :

- there is no 4.2 line
- 4.1 latest release is 4.2.5 
- 4.1 lacks an EOL date

And If have another question : according to https://docs.joomla.org/Release_and_support_cycle, _you should always use or update to the latest major version's minor.patch_. But according to https://docs.joomla.org/Joomla!_CMS_versions, version 4.0 is still supported with the EOL date 15.02.2024. Is this an error (and this EOL date should be for the latest 4.x version) or is it correct ?

**Comment by wilsonge** [comment]
Yes it was an error. Hopefully I've just fixed all the issues you've pointed out!

**Comment by marcwrobel** [comment]
Thanks !
